DEVELOPER.md

```diff
@@ -477,6 +477,8 @@ make smoke-test-k8
 Perform CLI smoke test against your running cluster (Kubernetes or local)
 
 ```
+export FLUVIO=<path to fluvio binary,e.g. target/debug/fluvio>
```

Comment:
prefer explicit working code instead of comment in the code block

Reply:
changed it to working code